guides/common/modules/proc_updating-smart-proxy.adoc

```diff
@@ -4,6 +4,9 @@
 Update {SmartProxyServers} to the next minor version.
 
 .Procedure
+. Synchronise the {RepoRHEL8ServerSatelliteCapsuleProjectVersion} repository in the {ProjectServer}.
+Publish and promote a new version of the content view with which the {SmartProxy} is registered.
```

Comment:
```suggestion
ifdef::satellite[]
. Synchronise the {RepoRHEL8ServerSatelliteCapsuleProjectVersion} repository in the {ProjectServer}.
Publish and promote a new version of the content view with which the {SmartProxy} is registered.
endif::[]
```

This attribute is only defined for satellite build. However; you will need a confirmation from SME that this step only applies to Satellite.

Reply:
ack. I am waiting for the SMEs to confirm this. However, I checked the installation doc as well and do not see any corresponding repo related changes there, so this might just be applicable to Satellite. Will update as soon as I get an ack.

Reply:
@adamlazik1 SMEs have confirmed that the changes apply only to Satellite and not to the other builds.